themes/darkpurple.json

```diff
@@ -0,0 +1,18 @@
+{
+    "meta_description":"A clean theme that highlights our purple nature",
```

Comment:
maybe it's too early to add metadata... I used it as comment here, because vanilla json supports no comments.